_docs/tutorial/06-add-istio.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Add istio
+overview: Overview
+
+order: 06
+
+layout: docs
+type: markdown
+---
+{% include home.html %}\n# In this module we will deploy the Istio control plane. We will enable Istio for a single microservice.
+
+As we saw in the previous module, Kubernetes does not provide us all the functionality we need for effective operating of our microservices. Istio comes to our help.
+
+First we deploy the _Istio control plane_. Then we enable Istio on a single microservice, _productpage_. The rest of the application will continue to operate as previously. Note that we can enable Istio gradually, microservice by microservice. Also note that Istio is enabled transparently to the microservices, we do not change the microservices code. And also note that we enable Istio without disrupting our application, it continues to run and serve user requests.
+
+We use `istiocl kube-inject` command to inject Istio _sidecar proxies_ into the microservice pods.
+
+1. Install Istio
+   ```bash
+   kubectl apply -f ../../istio-*/install/kubernetes/istio.yaml
+   ```
+1. Verify that Istio is started correctly, all the pods in `istio-system` namespace are running.
+   ```bash
+   kubectl get pods -n istio-system
+   ```
+1. Deploy Bookinfo application, Istio-enabled
+   ```bash
+   kubectl apply -f <(istioctl kube-inject -f ../03-run-bookinfo-with-kubernetes/bookinfo.yaml)
+   ```
+
+1. Access the application and verify that the application continues to work. Note that Istio was added **transparently**, the code of the original application did not change.
+
+2. Check the pods of the _productpage_ and see that now each replica has two containers. The first container is the microservice itself, the second is the sidecar proxy attached to it.
```

Comment:
Show commands and sample output?

Reply:
@geeknoid I will add `kubectl get pods`. Regarding the output, I have my doubts. In this tutorial I do not show any output to make it as short as possible, only the instructions to run. Originally, I used this tutorial for my demonstration of Istio. So this tutorial is for quick self-learning or for a quick demonstration of most of the Istio features. Let's think if we want to expand the tutorial with the output.

Reply:
@geeknoid I have second thoughts about the output, maybe I should add output as in tasks and blog posts.